### DISCUSSION TITLE
Create new custom drawer size

### DISCUSSION THREAD

**Discussion by netcoding87** [chakra-ui/chakra-ui#6977] [Q&A]
Hi there,

I am trying to add a new size to `Drawer` component to be used for one specific use case. Unfortunately I am unable to get it running... Based on https://chakra-ui.com/docs/styled-system/component-style#styling-multipart-components and this [issue](https://github.com/chakra-ui/chakra-ui/issues/3760) I tried to extend the theme, but it seems that the styles are not applied correctly... 

Does anyone of you can point me into the right direction? 

Codesandbox: https://codesandbox.io/s/loving-solomon-bh9zmc?file=/src/index.tsx

**Comment by netcoding87** [comment]
Was able to solve it on my own 🎉 The solution is to use `dialog` instead of `drawer` in theme extension :)

You can see a working example here: https://codesandbox.io/s/headless-thunder-sznnxe?file=/src/index.tsx:1244-1331

**Reply by auttmme** [reply]
@netcoding87 I stumbled upon the same problem and your solution works for me. Thank you so much!!
